src/components/ProofOfAddress/PoAIntro/index.tsx

```diff
@@ -8,19 +8,20 @@ import { POA_INTRO_LOCALES_MAPPING } from '~utils/localesMapping'
 import { trackComponent } from '../../../Tracker'
 import { StepComponentBaseProps } from '~types/routers'
 import style from './style.scss'
+import { CountryData } from '~types/commons'
 
 type Props = {
-  country?: string | undefined
+  poaDocumentCountry?: CountryData | undefined
 } & StepComponentBaseProps
 
-const PoAClientIntro: FunctionComponent<Props> = ({ country, nextStep }) => {
+const PoAClientIntro: FunctionComponent<Props> = ({ nextStep }) => {
   const { translate, parseTranslatedTags } = useLocales()
 
   return (
     <div className={theme.fullHeightContainer} data-page-id={'PoAIntro'}>
       <PageTitle
         title={translate('poa_intro.title', {
-          country: !country || country === 'GBR' ? 'UK' : '',
+          country: '',
```

Comment:
If we don't need the `%{country}` in the locale string, please remove them from there. Also update lokalise (software where we store our locales string)

Reply:
I'm not sure I can do the full update, as it doesn't appear that I have access to the lokalise software. But, I did make the code updates.

Reply:
I see that you added them. Thanks for the effort to include them!